backend/auth.js

```diff
@@ -0,0 +1,33 @@
+const jwt = require('jsonwebtoken');
+
+// define a middleware function to check authentication
+module.exports = (req, res, next) => {
+    // console.log("req only", req);
+    // console.log("req auth headers", req.headers['authorization']);
+    const authHeader = req.headers['authorization'];
+    console.log("authHeader", authHeader);
```

Comment:
delete this log, we dont need any sensitive data as such in prod

Reply:
Thanks, will remove it